manifest-schema.graphql

```diff
@@ -35,6 +53,16 @@ type EthereumContractMapping {
   eventHandlers: [EthereumContractEventHandler!]
 }
 
+type NearContractMapping {
+  apiVersion: String!
+  language: String!
+  file: File!
+  entities: [String!]!
+  blockHandlers: [NearBlockHandler!]
+  receiptHandlers: [NearReceiptHandler!]
+  functionCallHandlers: [NearFunctionCallHandler!]
```

Comment:
Lets comment out `functionCallHandlers` for now.

Reply:
⛔   **r e m o v e d** ⛔